ARE/components/actuator.fs20sender/src/main/java/eu/asterics/component/actuator/fS20Sender/FS20Utils.java

```diff
@@ -220,7 +201,14 @@ private static String getArgString(String[] strings) {
         return args.trim();
     }
 
-    static void copyBatchFromJar(String batchFilename) throws IOException {
+    /**
+     * copies a file with given name from java package to a temp-file. the temp file is deleted on exit of the program
+     *
+     * @param batchFilename
+     * @return the path of the temp-file
+     * @throws IOException
+     */
+    static String copyBatchFromJar(String batchFilename) throws IOException {
     	//This is a small hack because we want to get the system or user temp dir but don't want to get a unique tempfilename.
     	//Create a temp file 
     	File tempFile=File.createTempFile(batchFilename,"");
```

Comment:
so this line is not used any more?

The diff view is strange, it seems like there is some code missing.
Where is the variable 
batchFile initialized?

Anyway to autodelete the file on exit, you can also use

batchFile.deleteOnExit()

Reply:
you have to click here: http://prntscr.com/gzerwj to show everything. in diff-view only the diffs are shown.
there is already a batchFile.deleteOnExit() but it has no effekt - the file is not deleted on exit. but maybe this is because i'm killing the process in the console with Ctrl+C and this is not a proper exit?

Reply:
the line you commented is still here - it is white and has no changes. just red lines are removed, green ones are added.

Reply:
so you just added returning the absolute path.
-->
The delete works if you close the ARE normally with the GUI. 
So if this works then please merge!

Reply:
yes, exactly, i just return the path. perfect, i'll merge :)